src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -18,6 +18,12 @@ public abstract class AbstractConfigurationReader
 		protected internal static LogLevel ParseLogLevel(string value) =>
 			Enum.TryParse<LogLevel>(value, out var logLevel) ? logLevel : AbstractLogger.LogLevelDefault;
 
+		protected string ParseSecretToken(ConfigurationKeyValue kv)
+		{
+			if (kv == null || string.IsNullOrEmpty(kv.Value)) return null;
```

Comment:
I thought a little bit about maybe returning `String.Empty`, but I'm not sure... I bring this up, because prior to this change our configs never returned `null`, this would be the first one. Whether this is a real issue or not, I'm not sure... Maybe someone has a stronger opinion, I'm ok with this.

Reply:
I typically lean on evaluating `string.Empty` and `null` to `null`, but don't have strong opinions here.

Reply:
Yeah, I also think this is fine. 👍